scrollbar.kak

```diff
@@ -39,17 +34,6 @@ define-command update-scrollbar -hidden -override %{
     }
 }   
 
-# Launch / update the scrollbar highlighter
-define-command draw-scrollbar -hidden -override %{
-    addhl -override window/scrollbar-kak flag-lines Scrollbar scrollbar_flags
-}
-
-# Remove the scrollbar for this window
-define-command remove-scrollbar -hidden -override %{
-    rmhl window/scrollbar-kak
-    rmhooks window scrollbar-kak
-}
-
 # Move scrollbar to the left when necessary, as a user-activated command.
 define-command move-scrollbar-to-left -override -docstring %{
```

Comment:
Does this command need to be updated on account of the `draw-scrollbar` function being inlined?

Reply:
Of course, you're completely right. I didn't notice it myself because I don't normally use a left-column highlighter other than the scrollbar.